README.md

```diff
@@ -114,6 +114,30 @@ JsBarcode(".barcode").init();
 ![Result](http://imgh.us/upc.svg)
 
 
+
+#### Retrieve the barcode values so you can render it any way you'd like
+Pass in an object which will be filled with data.
+```
+const data = {};
+JsBarcode(data, 'text', {...options});
+```
+data will be filled with a ``` encodings ``` property which has all the needed values.
+
+Example:
+```
+{ encodings:
+   [ { data: '00000000', text: '5', options: [Object] },
+     { data: '1010110001', options: [Object], text: '' },
+     { data: '01100010001101000110100011010001101',
+       text: '50000',
+       options: [Object] },
+     { data: '01010', options: [Object], text: '' },
+     { data: '11100101110010111001011100101110010',
+       text: '00000',
+       options: [Object] },
+     { data: '1110010101', options: [Object], text: '' },
+     { data: '00000000', text: '0', options: [Object] } ] }
+```
```

Comment:
I think it might be a too specific case to describe in the README. This is better suited for the Wiki 😄 Move it there and remove this from the PR and I will merge 😃